### DISCUSSION TITLE
Дока — автор, советчик, контрибьютор

### DISCUSSION THREAD

**Discussion by furtivite** [doka-guide/content#2292] [Разное]
В Доке скопились статьи и мнения авторов (блок «На практике» — ex. «В работе»), которые писали до запуска и не пожелали публичности после. Эти статьи и советы были присвоены Доке.

Такой подход не очень подходит публичной и открытой энциклопедии.

**Что делать со статьями?**
Редакция приняла решение пометить эти статьи, как требующие улучшения, и любой автор, который внесёт значительные изменения, перепишет статью, станет новым и полноценным автором. 

Статьи помечены плейсхолдерами.

Необходимо решить, что делать с советами, кажется, среди них есть как полезные, так и не очень.

**Comment by nlopin** [comment]
Я думаю, нужно их отревьюить и выкинуть то, что нам больше не кажется ценным. А по оставшимся решить индивидуально. Я не вижу ничего страшного в том, чтобы присвоить их авторам статей, если авторы разделяют совет
